h/templates/accounts/developer.html.jinja2

```diff
@@ -0,0 +1,51 @@
+{% extends "h:templates/layouts/profile.html.jinja2" %}
+
+{% set page_route = 'profile_developer' %}
+{% set page_title = 'Developer' %}
+
+{% block page_content %}
+  <div class="form-vertical">
+    <h2 class="form-heading">
+      <span>{% trans %}Generate your API token{% endtrans %}</span>
+    </h2>
+
+    {% if token %}
+      <label for="token">{% trans %}Your API token is:{% endtrans %}</label>
+      <input id="token" type="text" value="{{ token }}" readonly="readonly"
+             class="developer-page__api-token-input">
+      <span class="help-block">
+        Please keep your API token safe as it can be used to access your
```

Comment:
Hmm. But it's not true that it can edit _all_ data. You change your user profile settings over the API, for example. It's just the annotations and I think maybe indirectly groups too. Also you can _create_ new annotations and groups. I'm not sure I want to keep this statement up to date to reflect exactly what an API token can and can't do as the API changes, so I went for something generic.